docs/getting-started/fundamentals/block-in-the-editor.md

```diff
@@ -0,0 +1,168 @@
+# The block in the Editor
+
+The Block Editor is a React Single Page Application (SPA) and every block in the editor is displayed through a React component defined in the `Edit` property of the settings object used to [register the block on the client](https://developer.wordpress.org/block-editor/getting-started/fundamentals/registration-of-a-block/#registration-of-the-block-with-javascript-client-side). 
```

Comment:
```suggestion
The Block Editor is a React Single Page Application (SPA) and every block in the editor is displayed through a React component defined in the `edit` property of the settings object used to [register the block on the client](https://developer.wordpress.org/block-editor/getting-started/fundamentals/registration-of-a-block/#registration-of-the-block-with-javascript-client-side). 
```
I think this should be `edit` 🤔 The `Edit` component is assigned to the `edit` property.